### ISSUE TITLE
Is it still working now?

### ISSUE THREAD

**Issue by cjnchk** [ardeora/hyper-dynamic-wallpaper#1]
hyper version: 3.2.3

**Comment by cjnchk** [comment]
Download the release one is not work, but building by myself is work.

**Comment by cjnchk** [comment]
Download the release one is not working, but building by myself is working.

**Comment by ardeora** [comment]
Hey there! Yes. It still works :D If you download the release from GitHub then you would still have to add the extension name in the `hyper.js` config like so 
```js
localPlugins: [
  'hyper-dynamic-wallpaper'
],
```
I'll correct the docs soon to include that step :D Let me know if it still doesn't work for you

**Comment by cjnchk** [comment]
> Hey there! Yes. It still works :D If you download the release from GitHub then you would still have to add the extension name in the `hyper.js` config like so
> 
> ```js
> localPlugins: [
>   'hyper-dynamic-wallpaper'
> ],
> ```
> 
> I'll correct the docs soon to include that step :D Let me know if it still doesn't work for you

Yes, I have added this config(include my friends), but still not working, don't no why.
ps: This plugin is very nice and hope add transparency config in the future.

**Comment by ardeora** [comment]
Thanks for the feedback! That's strange. I'll try to replicate the issue and make a fix soon. Also someone has already created a pull request to add transparency config 🥳 I'll merge that change soon
